templates/compositions/ec-component-organisation-teaser/organisation-teaser.html.twig

```diff
@@ -0,0 +1,117 @@
+{% spaceless %}
+
+{#
+/**
+ * @file
+ * Theme implementation for ECL Organisation Teaser.
+ *
+ * Available variables:
+ * - "name" (string) (default: ''): name of the organisation.
+ * - "acronym" (string) (default: ''): acronym of the organisation.
+ * - "logo" (ImageValueObject) (default: {}): logo of the organisation.
+ * - "organisation_details" (array) (default: []): list of fields.
+ * - "contact_page_url" (string) (default: ''): email adress or contact page to contact the organisation.
+ * - "contact_page" (array) (default: []): processed organisation contact page url.
+ */
+#}
+
+{# Internal properties #}
+
+{% set _css_class = 'ecl-content-item ecl-u-d-flex ecl-u-pb-m' %}
+{% set _extra_attributes = '' %}
+{% set _name = name|default('') %}
+{% set _acronym = acronym|default('') %}
+{% set _logo = logo|default({}) %}
+{% set _organisation_details = organisation_details|default([]) %}
```

Comment:
We shouldn't default empty values, Twig takes care of that. I understand we did that in the past, somewhere else,  that was supposed to be temporary as those compositions were supposed to be incorporated in ECL Twig. So let's not do this from now on, and just let Twig defaults to empty values.